pkg/services/govmomi/util.go

```diff
@@ -525,7 +533,14 @@ func waitForIPAddresses(
 			// If the device spec requires static IP addresses, the wait
 			// is not over if the device lacks one of those addresses.
 			for _, specIP := range deviceSpec.IPAddrs {
-				if _, ok := macToHasStaticIP[mac][specIP]; !ok {
+				ip, _, err := gonet.ParseCIDR(specIP)
+				if err != nil {
+					ctx.Logger.Info(
```

Comment:
shall we use log.Error() ?

Reply:
Error is probably better. It's also important to remember that these logs are not typically user-facing.

Reply:
I agree here